### PR TITLE
fix: add force topic

### PR DESCRIPTION
**Pull request by jshackell-sfdc**
### What does this PR do?

Adds "force" topic above the subtopics so that the CLI Ref is correctly generated. 

### What issues does this PR fix or reference?

@W-8720968@

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,11 +69,16 @@
       "@oclif/plugin-help"
     ],
     "topics": {
-      "limits": {
-        "description": "display current org’s limits",
+      "force": {
+        "external": true,
```
Comment (mshanemc) [inline package.json:73]: what does external:true do?

Comment (jshackell-sfdc) [reply package.json:73]: I don't know, I just copied the json snippet from other plugins where the CLi ref is working.  @amphro ?

Comment (mshanemc) [reply package.json:73]: this is correct.  Thomas explained it for me.

Comment (jshackell-sfdc) [reply package.json:73]: So what does it do?

File: package.json [hunk 2]
```diff
@@ -69,11 +69,16 @@
...
         "subtopics": {
-          "api": {
-            "description": "display current org’s api limits"
+          "limits": {
+          "description": "display current org’s limits",
+              "subtopics": {
+              "api": {
```
Comment (mshanemc) [inline package.json:78]: should be indented

**Review by mshanemc — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (jshackell-sfdc) [reply package.json:78]: Better?

File: package.json (hunk 1 above)
Comment (amphro) [reply package.json:73]: It says that the description is defined in another plugin so we don't have multiple plugins defining the same description. Maybe it should be renamed. It is only a property used by the command reference generator.

**Review by amphro — APPROVED** [review]

**Review by mshanemc — APPROVED** [review]

**Pull request merged**